src/main/java/config/HudConstants.java

```diff
@@ -35,6 +35,10 @@
     public static final Color BUTTONCOLOR = Color.GRAY;
     public static final Color TEXTCOLOR = Color.BLACK;
     public static final Color HPBARCOLOR = new Color(255, 0, 0, 100);
+    public static final String NO_ITEMS_PLACEHOLDER = "No items";
+    public static final int ATTACK_MENU_POS_IN_BATTLE_MENU = 0;
+    public static final int CATCH_MENU_POS_IN_BATTLE_MENU = 1;
+    public static final int ITEM_MENU_POS_IN_BATTLE_MENU = 4;
 
```

Comment:
Work with Position in ArrayList. The array ist in PlayerConfig and is call PLAYER_ACTIONS